userguide/tutorials/development.adoc

```diff
@@ -306,12 +334,12 @@ Once the build is successful, you can run the application to verify that everyth
 
 The steps above configure the application so that the Kill Bill logs are displayed on the console. You can however customise this to save the logs in a separate log file. In order to set this up, you need to follow the steps given below:
 
-. Modify the `PROJECT_ROOT/profiles/killbill/src/main/resources/logback.xml` as follows:
+. Modify the `PROJECT_ROOT/profiles/killbill/src/main/resources/logback.xml`. Add the following after the license section:
 
  <?xml version="1.0" encoding="UTF-8"?>
-<configuration>
+  <configuration>
    <jmxConfigurator />
-   <property name="LOGS_DIR" value="F:/killbill/logs" />
+   <property name="LOGS_DIR" value="<log_file_path>" />
```

Comment:
Maybe mention above that `<log_file_path>` needs to be updated

Reply:
This is already mentioned in the subsequent step